praw/models/reddit/redditor.py

```diff
@@ -301,6 +325,45 @@ def trophies(self) -> List["praw.models.Trophy"]:
         """
         return list(self._reddit.get(API_PATH["trophies"].format(user=self)))
 
+    def trust(self):
+        """Add the Redditor to your whitelist of trusted users.
+
+        Trusted users will always be able to send you PMs.
+
+        Example usage:
+
+        .. code-block:: python
+
+            reddit.redditor("AaronSw").trust()
+
+        Use the ``accept_pms`` parameter of :meth:`.Preferences.update` to toggle your
+        ``accept_pms`` setting between ``"everyone"`` and ``"whitelisted"``. For
+        example:
+
+        .. code-block:: python
+
+            # Accept private messages from everyone:
+            reddit.user.preferences.update(accept_pms="everyone")
+            # Only accept private messages from trusted users:
+            reddit.user.preferences.update(accept_pms="whitelisted")
+
+        You may trust a user even if your ``accept_pms`` setting is switched to
+        ``"everyone"``.
+
+        .. note::
+
+            You are allowed to have a user on your blocked list and your friends list at
+            the same time. However, you cannot trust a user who is on your blocked list.
+
+        .. seealso::
+
+            - :meth:`.distrust`
+            - :meth:`.Preferences.update`
+            - :meth:`.trusted`.
```

Comment:
```suggestion
            - :meth:`.trusted`
```
Sorry I missed this.

Reply:
You're welcome to commit things like this before squash-and-merging. I always keep the "Allow edits from maintainers" box checked.